assets/js/bespoke.js

```diff
@@ -0,0 +1,9 @@
+window.onclick = function () {
+  var $addAnothers = document.querySelectorAll('[data-module="moj-add-another"]')
+  const addAnother = new MOJFrontend.AddAnother($addAnothers)
+  const buttons = document.getElementsByClassName('delete-condition-button')
+  if (buttons.length) {
+    var button = buttons[0]
+    button.textContent = addAnother.getItems().length > 1 ? 'Delete these conditions' : 'Delete this condition'
+  }
+}
```

Comment:
would be good to add a cypress test for this if possible as this is tightly coupled to the internal implementation of the add another component (ideally we'd ask them to add some hooks into their API).

A cypress test would help ensure it doesn't break during updates to the library - I think it would be very easy to miss, during a manual regression test, if this behaviour was broken